Library/Homebrew/brew.rb

```diff
@@ -105,7 +105,7 @@
     possible_tap = OFFICIAL_CMD_TAPS.find { |_, cmds| cmds.include?(cmd) }
     possible_tap = Tap.fetch(possible_tap.first) if possible_tap
 
-    if possible_tap && !possible_tap.installed?
+    if !possible_tap&.installed?
```

Comment:
Is this different from `unless possible_tap&.installed?` not work?

Reply:
It's the same but it, like me, doesn't like `unless ... else`

Reply:
ah